lib/element.rb

```diff
@@ -113,16 +122,43 @@ def click
     end
   end
 
+  #
+  # add to what's already in the text field
+  # for cases when you don't want to stomp what's already in the text field
+  #
+
+  def append_keys(*args)
+    ElementExtensions.highlight(self) if Gridium.config.highlight_verifications
+    $verification_passes += 1
+    unless element.enabled?
+      raise "Browser Error: tried to enter #{args} but the input is disabled"
+    end
+    element.send_keys(*args)
+    sleep @text_padding_time
+    # when it's possible to validate for more than non-empty outcomes, do that here
+  end
+
+  #
+  # overwrite to what's already in the text field
+  # and validate afterward
+  #
+
   def send_keys(*args)
-    Log.debug("Typing: #{args} into element: (#{self}).")
-    if element.enabled?
```

Comment:
Will this have any adverse effects if we try and highlight an element that is disabled?